tools/llm_bench/task/speech_to_text_generation.py

```diff
@@ -0,0 +1,188 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2023-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+import os
+import time
+import datetime
+import numpy as np
+from pathlib import Path
+import hashlib
+import logging as log
+import llm_bench_utils
+import llm_bench_utils.model_utils as model_utils
+import llm_bench_utils.metrics_print as metrics_print
+import llm_bench_utils.gen_output_data as gen_output_data
+import llm_bench_utils.parse_json_data as parse_json_data
+from llm_bench_utils.hook_forward_whisper import WhisperHook
+
+FW_UTILS = {'pt': llm_bench_utils.pt_utils, 'ov': llm_bench_utils.ov_utils}
+whisper_hook = WhisperHook()
+
+DEFAULT_OUTPUT_TOKEN_SIZE = 1000
+
+
+def run_speech_2_txt_generation(input_param, args, md5_list, iter_data_list):
+    result_md5_list = []
+    max_rss_mem_consumption = ''
+    max_uss_mem_consumption = ''
+    max_shared_mem_consumption = ''
+    pipe = input_param['pipe']
+    raw_speech = input_param['raw_speech']
+    num = input_param['iter_idx']
+    speech_id = input_param['speech_idx']
+    mem_consumption = input_param['mem_consumption']
+    processor = input_param['processor']
+    use_genai = input_param['use_genai']
+    speech_language = input_param['speech_param'].get('language', "<|en|>")
+    ret_timestamps = input_param['speech_param'].get('timestamp', True)
+    max_gen_tokens = DEFAULT_OUTPUT_TOKEN_SIZE if args['infer_count'] is None else args['infer_count']
+
+    if (args['mem_consumption'] == 1 and num == 0) or args['mem_consumption'] == 2:
+        mem_consumption.start_collect_memory_consumption()
+    if use_genai:
+        start = time.perf_counter()
+        result_text = pipe.generate(
+            raw_speech,
+            max_new_tokens=max_gen_tokens,
+            # 'task' and 'language' parameters are supported for multilingual models only
+            language=speech_language,
+            task="translate",
```

Comment:
Should task be set to `transcribe` as it's default option?

Reply:
@as-suvorov  Should the task of both the genai API and optimum-intel API set to 'transcribe', or is only the genai API set to 'transcribe'?